js/src/framework.ts

```diff
@@ -673,8 +688,9 @@ async function runEvaluatorInternal(
         },
       });
     }
-  });
-  const results = await Promise.all(evals);
+  }, evaluator.maxConcurrency);
```

Comment:
looks like based on the async.queue impl, if concurrency is omitted (functionally the same as setting it explicitly to undefined), the concurrency gets set to 1: https://github.com/caolan/async/blob/master/lib/queue.js#L104

might need to change this to the following to preserve current behavior:

```suggestion
  }, evaluator.maxConcurrency ?? data.length);
```

Reply:
Fixed! Thanks!